forum/users/serializers.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework import serializers
+from .models import CustomUser
+from django.core.validators import validate_email
+from django.contrib.auth.password_validation import validate_password
+from django.core.exceptions import ValidationError
+
+
+class CustomUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
+        fields = [
+            "user_id",
+            "first_name",
+            "last_name",
+            "email",
+            "user_phone",
+            "role",
+            "title",
+            "password",
+            "created_at",
+            "updated_at",
+        ]
+        read_only_fields = ("created_at", "updated_at")
+        extra_kwargs = {"password": {"write_only": True}}
+
+    def create(self, validated_data):
+        password = validated_data.pop("password")
+        instance = CustomUser.objects.create(**validated_data)
+        instance.set_password(password)
+        instance.save()
+        return instance
```

Comment:
When no password is provided during creation, `set_password` might raise an issue. Ensure a proper check before saving.